python-package/xgboost/plotting.py

```diff
@@ -184,18 +184,16 @@ def to_graphviz(booster, fmap='', num_trees=0, rankdir='UT',
     no_color : str, default '#FF0000'
         Edge color when doesn't meet the node condition.
     condition_node_params : dict (optional)
-        condition node configuration,
```

Comment:
Corresponds to unexpected indent and something like code block ending without a new line.. The former is still here.